File: delivery-system-server/src/main/java/ru/delivery/system/model/json/UserInfoResponse.java
package ru.delivery.system.model.json;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;


public class UserInfoResponse {
    @JsonProperty("id")
    @Getter @Setter
    private Integer id;

    @JsonProperty("username")
    @Getter @Setter
    private String username;

    @JsonProperty("name")
    @Getter @Setter
    private String name;

    @JsonProperty("surname")
    @Getter @Setter
    private String surname;

    @JsonProperty("role")
    @Getter @Setter
    private String role;

    @JsonProperty("lastLatitude")
    @Getter @Setter
    private Float lastLatitude;

    @JsonProperty("lastLongitude")
    @Getter @Setter
    private Float lastLongitude;
}

File: delivery-system-desktop/src/main/kotlin/ru/delivery/system/models/json/UserInfoResponse.kt
package ru.delivery.system.models.json

import com.fasterxml.jackson.annotation.JsonProperty

class UserInfoResponse {
    @JsonProperty("id")
    var id: Int? = null

    @JsonProperty("username")
    var username: String? = null

    @JsonProperty("name")
    var name: String? = null

    @JsonProperty("surname")
    var surname: String? = null

    @JsonProperty("role")
    var role: String? = null

    @JsonProperty("lastLatitude")
    var lastLatitude: Float? = null

    @JsonProperty("lastLongitude")
    var lastLongitude: Float? = null
}
